backend/presentation/views.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.viewsets import ModelViewSet
+
+from presentation.models import Comment
+from presentation.permissions import IsAdminOrOwner
+from presentation.serializers import CommentListSerializer, CommentSerializer
+from presentation.services import CommentService
+
+
+class CommentViewSet(ModelViewSet):
+    queryset = Comment.objects.all()
+    serializer_class = CommentListSerializer
+    permission_classes = [
+        IsAdminOrOwner,
+    ]
+    http_method_names = ["get", "post", "put", "patch"]
+
+    def perform_create(self, serializer):
+        serializer.save(user=self.request.user)
```

Comment:
You have created a service responsible for comments, move this part to the service.